frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/storage-tab-state-update.ts

```diff
@@ -137,7 +145,34 @@ export const internalStorageDiskBusUpdater = ({
   }
 };
 
+const windowsToolsUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
```

Comment:
it should be declared/called before the internalStorageDiskBusUpdater so the validation to it applies as well

Reply:
moved to before internalStorageDiskBusUpdater

Reply:
please move the code block as well. We are striving for our updaters to be declared in the same order they are executed in. It is much easier to read the file from top to the bottom to asses what is happening in sequence, than jumping across different sections of the file.

There are few exceptions (eg prefillVmTemplateUpdater) where the updater would get too long and such updaters are moved to new files.